algosdk/v2client/indexer.py

```diff
@@ -250,6 +250,176 @@ def account_info(
 
         return self.indexer_request("GET", req, query, **kwargs)
 
+    def account_asset_info(
+        self,
+        address,
+        limit=None,
+        next_page=None,
+        asset_id=None,
+        block=None,
+        round_num=None,
+        include_all=False,
+        **kwargs
+    ):
+        """
+        Return asset information for a specific account.
+
+        Args:
+            address (str): account public key
+            limit (int, optional): maximum number of results to return
+            next_page (str, optional): the next page of results; use the next
+                token provided by the previous results
+            asset_id (int): include transactions for the specified
+                asset
+            block (int, optional): use results from the specified round
+            round_num (int, optional): alias for block; only specify one of
+                these
+            include_all (bool, optional): include all items including closed
+                accounts, deleted applications, destroyed assets, opted-out
+                asset holdings, and closed-out application localstates. Defaults
+                to false.
+        """
+        req = "/accounts/" + address + "/assets"
+        query = dict()
+        _specify_round(query, block, round_num)
+        if include_all:
+            query["include-all"] = include_all
+        if limit:
+            query["limit"] = limit
+        if next_page:
+            query["next"] = next_page
+        if asset_id:
+            query["asset-id"] = asset_id
+
+        return self.indexer_request("GET", req, query, **kwargs)
+
+    def account_asset_by_creator(
```

Comment:
how did you come up with this name? The spec calls this function `lookupAccountAssets`, so I would expect the name to somehow be derived from that.